openaddr/process_one.py

```diff
@@ -157,7 +164,9 @@ def get_log_handler(directory):
     handler = logging.FileHandler(filename)
     handler.setFormatter(logging.Formatter(u'%(asctime)s %(levelname)08s: %(message)s'))
     handler.setLevel(logging.DEBUG)
-    handler.addFilter(LogFilter())
+    
+    # # Limit log messages to the current thread
+    # handler.addFilter(LogFilterCurrentThread())
```

Comment:
I did — it’s an old piece of code from 2015, and I think it might only have been there for a old multi-threaded version of this process. It was confusing me to not see some of the log messages go by.